validation/pipeline/tfp/Jenkinsfile.airgap.tests

```diff
@@ -0,0 +1,180 @@
+                              string(credentialsId: 'AWS_SSH_RSA_KEY', variable: 'AWS_SSH_RSA_KEY'),
+                              string(credentialsId: 'AWS_SSH_KEY_NAME', variable: 'AWS_SSH_KEY_NAME'),
+                              string(credentialsId: 'RANCHER_REGISTRY_USER_NAME', variable: 'RANCHER_REGISTRY_USER_NAME'),
+                              string(credentialsId: 'RANCHER_REGISTRY_PASSWORD', variable: 'RANCHER_REGISTRY_PASSWORD'),
+                              string(credentialsId: 'ADMIN_PASSWORD', variable: 'ADMIN_PASSWORD'),
+                              string(credentialsId: 'USER_PASSWORD', variable: 'USER_PASSWORD'),
+                              string(credentialsId: 'RANCHER_VALID_TLS_CERT', variable: 'RANCHER_VALID_TLS_CERT'),
+                              string(credentialsId: 'RANCHER_VALID_TLS_KEY', variable: 'RANCHER_VALID_TLS_KEY')]) {
+          
+                withEnv(paramsMap) {
+
+
+                    stage('Checkout Infrastructure Repo') {
+                        deleteDir()
+                        checkout([
+                                    $class: 'GitSCM',
+                                    branches: [[name: "*/${infraBranch}"]],
+                                    extensions: scm.extensions + [[$class: 'CleanCheckout']],
+                                    userRemoteConfigs: infraRepo
+                                ])
+                     }
+
+                    stage('Configure and Build') {
+                        config = config.replace('${AWS_SECRET_ACCESS_KEY}', env.AWS_SECRET_ACCESS_KEY)
+                        config = config.replace('${AWS_ACCESS_KEY_ID}', env.AWS_ACCESS_KEY_ID)
+                
+                        writeFile file: filename, text: config
+                
+                        dir(".ssh") {
+                            def decoded = new String(env.AWS_SSH_PEM_KEY.decodeBase64())
+                            writeFile file: AWS_SSH_KEY_NAME, text: decoded
+                    
+                            def decodedRsa = new String(AWS_SSH_RSA_KEY.decodeBase64())
+                            writeFile file: JENKINS_RKE_VALIDATION, text: decodedRsa
+                        }
+                        
+                        env.CATTLE_TEST_CONFIG=rootPath+filename
+                
+                        sh "./configure.sh"
+                        sh "./build.sh"
+                     }
+
+                    stage('Run Infrastructure Module Test') {
+                        try {
+                            sh """
+                            docker run --name ${testContainer} -t --env-file ${envFile} ${imageName} sh -c "
+                            /root/go/bin/gotestsum --format standard-verbose --packages=${testsDir} --junitfile ${testResultsOut} --jsonfile ${testResultsJSON} -- -timeout=${timeout} -v ${params.TEST_CASE} | tee /root/test_logs.txt"
```

Comment:
I find test_logs.txt weird, is it necessary or at least could it have a better name if it is necessary? as is, it sounds like it could be removed but I see later on we actually use grep on this log file to output stuff we need.